cpg_pipes/providers/cpg/status.py

```diff
@@ -88,7 +88,8 @@ def create_analysis(
         analysis_type: str,
         analysis_status: str,
         target: Target,
-        meta: dict | None = None,        
+        meta: dict | None = None,
+        project_name: str=None,
```

Comment:
I think the linter is complaining here 
```suggestion
        project_name: str = None,
```

Reply:
Addressed